zrml/hybrid-router/src/lib.rs

```diff
@@ -122,13 +125,27 @@ mod pallet {
     {
         /// A trade was executed.
         HybridRouterExecuted {
+            /// The type of transaction (Buy or Sell).
```

Comment:
Are the fee amounts contained in the `amount_*` values? This should be clarified in the documentation.

Reply:
2e58bedbc7609d4cb833c2f2501974fc1d729ec8